stdlib/public/core/LazySequence.swift

```diff
@@ -44,7 +44,7 @@
 ///       ///
 ///       ///     (1..<6).scan(0, +) // [0, 1, 3, 6, 10, 15]
 ///       ///
-///       /// - Complexity: O(N)
+///       /// - Complexity: O(n)
```

Comment:
This one won't be rendered, since it's a comment in a code sample, so I left it "plain text".